axelor-account/src/main/resources/views/Move.xml

```diff
@@ -671,7 +671,6 @@
     <action name="action-move-method-set-moveline-dates"/>
     <action name="action-move-attrs-date-change"/>
     <action name="action-move-method-display-and-compute-due-date"/>
-    <action name="action-account-move-group-onsave"/>
```

Comment:
You need to call every action in action-account-move-group-onsave that is linked to the Move.date
Per example, I think the action "action-move-method-update-invoice-terms" uses Move.date or even "action-move-method-update-due-date".